k8s/examples/elasticsearch/20-es-coordinator.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: es-coordinator
+  # namespace: elasticsearch
+  labels:
+    component: elasticsearch
+    role: coordinator
+spec:
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        kubernetes.io/change-cause: ""
+      labels:
+        component: elasticsearch
+        role: coordinator
+    spec:
+      serviceAccountName: elasticsearch
+      containers:
+        - name: es-coordinator
+          image: elasticsearch:6.7.0
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: "node.name"
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: "cluster.name"
+              value: "es"
+            - name: "node.master"
+              value: "false"
+            - name: "node.ingest"
+              value: "false"
+            - name: "node.data"
+              value: "false"
+            - name: "http.enabled"
+              value: "true"
+            - name: "discovery.zen.ping.unicast.hosts"
+              value: "es-discovery"
+            - name: ES_JAVA_OPTS
+              value: -Xms2048m -Xmx2048m -XX:ErrorFile=/dev/stderr
+            - name: "processors"
+              valueFrom:
+                resourceFieldRef:
+                  resource: limits.cpu
+            - name: "network.host"
+              value: "_eth0:ipv4_"
```

Comment:
This interfaces are not enough if you want to port-forward. I added the `_site_, _lo_` to make the port forward work in https://github.com/storageos/deploy/pull/53/files